packages/agent/src/providers/AgentProvider.tsx

```diff
@@ -15,7 +16,9 @@ export const AgentProvider = ({ agent, children }: PropsWithChildren<AgentProvid
   <NativeAgentProvider agent={agent}>
     <W3cCredentialRecordProvider agent={agent}>
       <SdJwtVcRecordProvider agent={agent}>
-        <ExchangeRecordDisplayMetadataProvider>{children}</ExchangeRecordDisplayMetadataProvider>
+        <SeedCredentialProvider agent={agent}>
```

Comment:
would be really weird that paradym wallet imports @ausweis/storage to add seed credential provider. This needs to go in ausweis wallet